tox/session.py

```diff
@@ -143,7 +142,7 @@ def popen(self, args, cwd=None, env=None, redirect=True, returnout=False, ignore
         elif returnout:
             stdout = subprocess.PIPE
         if cwd is None:
-            # XXX cwd = self.session.config.cwd
+            # FIXME XXX cwd = self.session.config.cwd
```

Comment:
this is not to descriptive what the fix me should be, maybe create an issue and link it to it?